tests/blockchain/eth/clients/test_mocked_clients.py

```diff
@@ -126,3 +202,65 @@ def test_ganache_web3_client():
     assert interface.client.platform is None
     assert interface.client.backend == 'ethereum-js'
     assert interface.client.is_local
+
+
+def test_datetime_logic():
```

Comment:
@michwill have a look at this test.  I think it proves that this datetime logic works as I originally assumed

Reply:
@vepkenez I think if you change the test, you might get a different result - which is what @michwill is pointing out:

```
In [19]: now = datetime.datetime.now()

In [20]: now
Out[20]: datetime.datetime(2019, 8, 8, 8, 36, 14, 323883)

In [21]: past_date = now - datetime.timedelta(days=3,seconds=3)

In [22]: past_date
Out[22]: datetime.datetime(2019, 8, 5, 8, 36, 11, 323883)

In [23]: (now - past_date).seconds
Out[23]: 3
```

Reply:
Using `total_seconds()` may be the way to go:
```
In [26]: (now - past_date).total_seconds()
Out[26]: 259203.0
```

Reply:
Ah yes.  I just got lucky there... total_seconds it is!

Reply:
That's another way, yes.
Although it doesn't quite matter: why datetime objects at all!